### PR TITLE
Updating for Visual Studio 2022

### PR DESCRIPTION
**Pull request by rprouse**
Continuing work on #43 properly updating the references and updating to the newer framework and adapter releases. Still needs testing. We may need to release a new version just for 2022 now that it is x64?

### CODE REVIEW

**Comment by rprouse** [comment]
TODO, follow https://learn.microsoft.com/en-us/visualstudio/extensibility/migration/update-visual-studio-extension?view=vs-2022

**Comment by OsirisTerje** [comment]
As written in the link you point to, we can create two installation targets for this extension so that it continues to work for VS2017/19.  I believe that VS2017/19 will complain if we release this for VS2022 - they might say it can be updated since it is already installed, although it will not work since this new one will be for x64.  So if we continue backward compatibility with 2017/19 we should have two installation targets.  Not sure how that will affect the other changes though. 

UPDATE:  Just noticed you had given the extension a new identifier, so then it will not crash.  That is probably for the best, but means we don't expect any support to the older ones.

File: CreateUnitTests.NUnit/NUnit2SolutionManager.cs [hunk 1]
```diff
@@ -39,6 +39,7 @@ public class NUnit2SolutionManager : SolutionManagerBase
     {
         private const string NUnitVersion = "2.6.4";
         private const string NunitAdapterVersion = "2.2.0";
```
Comment (OsirisTerje) [inline CreateUnitTests.NUnit/NUnit2SolutionManager.cs:41]:
```suggestion
        private const string NunitAdapterVersion = "2.3.0";
```

File: CreateUnitTests.NUnit/NUnit2SolutionManager.cs [hunk 2]
```diff
@@ -39,6 +39,7 @@ public class NUnit2SolutionManager : SolutionManagerBase
     {
         private const string NUnitVersion = "2.6.4";
```
Comment (OsirisTerje) [inline CreateUnitTests.NUnit/NUnit2SolutionManager.cs:40]:
```suggestion
        private const string NUnitVersion = "2.7.1";
```

**Review by OsirisTerje — APPROVED** [review]

**Comment by OsirisTerje** [comment]
@rprouse 
1.  Another thing, not sure if we should continue the support of NUnit 2 ?
2. You mention needing more testing - what is needed, or what is, or not, been tested?
3. We need to update the readme

**Comment by rprouse** [comment]
- I agree about dropping NUnit 2. I haven't even tested it. 
- I've only done some light testing on it so far. I'd like to do some more.
- Dropping support for old versions; we haven't updated this in a long time and I don't think it works properly in 2019. Cut our losses?

**Comment by rprouse** [comment]
@OsirisTerje I am going to merge this and remove NUnit2 in another PR to keep them separate.

**Pull request merged**